lib/commands/plugin-test.sh

```diff
@@ -39,7 +39,7 @@ plugin_test_command() {
     fi
 
     TEST_DIR=$(mktemp -dt asdf.XXXX)
-    git clone "$ASDF_DIR/.git" "$TEST_DIR"
+    cp -R "$ASDF_DIR/" "$TEST_DIR"
```

Comment:
These two commands are not equivalent Git clone only copies over the files that are checked in. It is has ignore rules for directories like `installs`, `plugins`, and `shims`.

Reply:
Running this locally it tries to copy everything in `~/.asdf`, which is a lot of install and plugin data.

Reply:
I'd suggest copying over just the things we need. `bin`, `lib`, `test`, etc...

Reply:
Yup, that was my fear. brew installs don't have $ASDF_DIR and $ASDF_DATA_DIR in the same spot. I'll make that change when I have the chance.